org.civicrm.reqangular/test/services/api_test.js

```diff
@@ -0,0 +1,122 @@
+define([
+  'common/angular',
+  'common/angularMocks',
+  'common/services/api'
+], function () {
+  'use strict';
+
+  describe('api test', function () {
+    var api, $httpBackend, $rootScope;
+
+    beforeEach(module('common.apis'));
+
+    beforeEach(inject(function (_api_, _$httpBackend_, _$rootScope_) {
+      api = _api_;
+      $rootScope = _$rootScope_;
+      $httpBackend = _$httpBackend_;
+    }));
+
+    describe('sendGET', function () {
+      var entity = 'send_get_entity',
+        action = 'send_get_action',
+        params = {key: 'value'},
+        cache = true,
+        promise;
+
+      afterEach(function () {
+        $httpBackend.flush();
+      });
+
+      describe('when there is no error', function () {
+        var returnValue = {
+          is_error: 0,
+          somekey: 'someval'
+        };
+
+        beforeEach(function () {
+          $httpBackend.whenGET(/action\=send_get_action&entity\=send_get_entity/)
+            .respond(returnValue);
+
+          promise = api.sendGET(entity, action, params, cache);
+        });
+
+        it('proper value is returned', function () {
```

Comment:
anyway this title is not clear. What is a "proper" value? are we implying that there is an "improper" value? aren't you simply saying that when there is no error the api service returns the data as is?